.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
pls add .vscode to .gitignore ; we don't want to check-in any personal ide preferences and settings.

Reply:
Hi @codecakes ,

my bad....totally oversaw that.  Will fix now!

Reply:
this file is still here. try git rm --cached <this_file>